src/@types/graphql.ts

```diff
@@ -0,0 +1,1192 @@
+export type Maybe<T> = T | null;
```

Comment:
Just for my own understanding, we are basically defining types for all our graphql requests and responses? Was this auto-generated?

Reply:
So far these types aren't actually being used anywhere right? This PR just setups the generation of these types right?

Reply:
Currently, they're not. However, they will be used to describe expected input/output data in our resolvers (see #196)